benchmark/syclblas/main.cpp

```diff
@@ -13,29 +13,35 @@ void free_device_selector() { cdsp.reset(); }
 ExecutorPtr Global::executorInstancePtr;
```

Comment:
I think now we should be able to remove global variable

Reply:
Oh yes I forgot about the executor, sorry.

Reply:
Fixed